docs/reference/runtimes/python/python-reference.md

```diff
@@ -23,6 +23,28 @@ def handler(context: nuclio_sdk.Context, event: nuclio_sdk.Event):
 
 The `handler` field is of the form `<package>:<entrypoint>`, where `<package>` is a dot (`.`) separated path (for example, `foo.bar` equates to `foo/bar.py`) and `<entrypoint>` is the function name. In the example above, the handler is `main:handler`, assuming the file is named `main.py`.
 
+For asynchronous support (e.g.: `asyncio`), you may want to decorate your function handle with `async`
+
+Do note that:
+  - Nuclio, at the moment, does not support concurrent requests handling for a single working. Each working may handle
+    one request at a time, for more information and reading further, see [here](/docs/concepts/architecture.md#runtime-engine).
+  - The event loop would keep running while listening on more incoming requests, that can benefit functions that do
```

Comment:
```suggestion
  - However using an async handler can still be beneficial in some scenarios; Since the event loop would keep running while listening on more incoming requests, it allows functions to asynchronously perform
```